lms/static/js/sysadmin/main.js

```diff
@@ -0,0 +1,25 @@
+function confirmAndSubmit(e){
```

Comment:
We should wrap all of this file in an IIFE to not pollute the global namespace here. 

```javascript
(function () {
    // your code here
}());
```

If you're unfamiliar with this practice, give a holler and I can walk you through it